setup.py

```diff
@@ -4,29 +4,44 @@
 
 
 INSTALL_REQUIREMENTS = [
-    "Django>=1.11,<2.0",
-    "django-cms>=3.5.0",
-    "django-treebeard>=4.3",
+    'Django>=1.11,<2.0',
+    'django_cms',
+    'django-treebeard>=4.3',
+]
+
+TEST_REQUIREMENTS = [
+    'djangocms_helper',
+    'djangocms_versioning',
+    'djangocms_version_locking',
+    'djangocms-moderation',
+    'factory_boy'
 ]
 
 
 setup(
-    name="djangocms-navigation",
+    name='djangocms-navigation',
```

Comment:
This is good to go bar the changes to the quotes. Once this is reversed these changes can be merged.